src/helpers/replaceArray.ts

```diff
@@ -0,0 +1,13 @@
+interface ReplaceArrayDictionary {
+	[key: string]: string;
+}
+
+type ReplaceArray = (str: string, find: ReplaceArrayDictionary) => string;
```

Comment:
Remove this `type`

Reply:
why? do you wanna make the `ReplaceArrayDictionary` and `ReplaceArray` a single type?